site/en/blog/migrate-way-from-data-urls-in-svg-use/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Migrate away from data URLs in SVG <use> element
+description: >
+  Support for data: URLs in SVG <use> element will be removed. We recommended to migrate to alternatives.
+date: 2023-05-25
+authors:
+  - shhnjk
+tags:
+  - security
+---
+
+{% Aside %}
+We expect to [remove support for `data:` URLs in SVG `<use>` element](https://chromestatus.com/feature/5128825141198848)
+in Chrome 119, scheduled to ship in November 2023.
+{% endAside %}
+
+The SVG spec was [recently updated](https://github.com/w3c/svgwg/pull/901) to remove support for `data:` URLs in SVG `<use>` element.
+This improves security of the Web platform as well as compatibility between browsers as Webkit does not support `data:` URLs in SVG `<use>` element.
```

Comment:
I'd be better to add "why to remove it" here. For example, share known incidents with summary, how to attack with data with poc code etc

Reply:
+1
I'm inclined to move the reasoning forefront.
Also, can you add the status in Firefox as well?

Reply:
Done :)
Since I don't know the status of Firefox (i.e. they are waiting for our successfully removal), I've linked thier standard position issue.